src/manage_migrators.c

```diff
@@ -1701,6 +1738,7 @@ static migrator_t database_migrators[] = {
   {205, migrate_204_to_205}, // v8.0: rev 205
   {206, migrate_205_to_206},
   {207, migrate_206_to_207},
+  {208, migrate_207_to_208},
```

Comment:
GVMD_DATABASE_VERSION in CMakeLists.txt needs to be updated for this to run.

Reply:
Fixed.